src/components/PersonDetails.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[];
+  person: Person;
+  selectedSlug: string;
+};
+
+export const PersonDetails:FC<Props> = ({ people, person, selectedSlug }) => {
+  const {
+    sex, born, died, motherName, fatherName, slug,
+  } = person;
+
+  const motherFound = people.find(item => item.name === motherName);
+  const motherCell = motherName || '-';
```

Comment:
Consider using 'motherFound?.name' instead of 'motherName' for 'motherCell'. This ensures that the value is '-' when 'motherFound' is undefined.

Reply:
I tried to use motherFound?.name instead of 'motherName' for 'motherCell' but tests were failed in this case. I receive '-' for all cases where motherFound is undefined. I didn't add this change to the code.